src/urllib3/response.py

```diff
@@ -317,26 +338,24 @@ def readable(self) -> bool:
 
     def readinto(self, b: bytearray) -> int:
         temp = self.read(len(b))
-        if len(temp) == 0:
+        if temp is None or len(temp) == 0:
```

Comment:
Since `read()` now returns only `bytes` can we remove the `is None` check here?

Reply:
done